api/tests/prescription/campaign/integration/infrastructure/repositories/campaign-to-join-repository_test.js

```diff
@@ -50,6 +52,26 @@ describe('Integration | Repository | CampaignToJoin', function () {
       expect(actualCampaign.identityProvider).to.equal(OidcIdentityProviders.POLE_EMPLOI.code);
     });
 
+    it('should return restricted access if organization has learner import feature', async function () {
+      // given
+      const organization = databaseBuilder.factory.buildOrganization({ isManagingStudents: false });
```

Comment:
**To do :** une fois l'injection de dépendance faite, nous n'aurons pas besoin de "re-"tester cette partie là donc nous n'aurons pas besoin de recréer une organization ou une featureId